net-misc/gerbera/gerbera-2.1.0.ebuild

```diff
@@ -0,0 +1,90 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit cmake linux-info
+
+if [[ ${PV} == 9999 ]]; then
```

Comment:
Try to respect the order from `skel.ebuild` if possible.  Move this whole if block below `HOMEPAGE`